src/shared/wifi-util.c

```diff
@@ -23,6 +23,13 @@ int wifi_get_interface(sd_netlink *genl, int ifindex, enum nl80211_iftype *iftyp
                 return log_debug_errno(r, "Could not append NL80211_ATTR_IFINDEX attribute: %m");
 
         r = sd_netlink_call(genl, m, 0, &reply);
+        if (r == -ENODEV) {
+                /* For obsolete WEXT driver. */
+                log_debug("Failed to request information about wifi interface %d. "
```

Comment:
doesn't matter much, but I#d allways include the error code here as in `log_debug_errno(r, …` — even if `%m` is not used, since this is about a single fixed error code only. That's because log_debug_errno() generates structured logs where the passed code is included, and it might be useful to see the code in the structured logs ...